### PR TITLE
Phone keypad

### PR DESCRIPTION
**Pull request by ishwari20**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Given an integer n, using phone keypad find out and print all the possible strings that can be made using digits of input n.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #4668

### CODE REVIEW

File: C-Plus-Plus/other/Phone_keypad.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
+
+#include <iostream>
+#include <string>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Phone_keypad.cpp]: Add one single header file, i.e **bits/stdc++.h** and remove the these two headers

File: C-Plus-Plus/other/Phone_keypad.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    Given an integer number print all the possible combinations of the keypad. You do not need to return anything just print them.
+    */
+    
+    string output = "";
+    printKeypad_rec(num, output);
+}
+
+int main(){
+    int num;
+    cin >> num;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Phone_keypad.cpp]: Display message in print form as what are you taking input from the user

File: C-Plus-Plus/other/Phone_keypad.cpp [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    printKeypad(num);
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Phone_keypad.cpp]: Display message in print form as what are you showing output to the user for better UI

File: C-Plus-Plus/other/Phone_keypad.cpp [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+ea
+fa
+db
+eb
+fb
+dc
+ec
+fc
+
+Time complexity: O(4n)
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Phone_keypad.cpp]: Time Complexity is **Wrong**
It should be  **O(4^n)**

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@ishwari20 Do the changes as suggested

**Comment by ishwari20** [comment]
@siddharth25pandey I have made those changes

File: C-Plus-Plus/other/Phone_keypad.cpp [hunk 5]
```diff
@@ -0,0 +1,67 @@
+/*
+Given an integer n, using phone keypad find out and print all the possible strings that can be made using digits of input n.
+*/
+#include<bits.stc++.h>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Phone_keypad.cpp]: Spelling Error

File: C-Plus-Plus/other/Phone_keypad.cpp [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    printKeypad_rec(num, output);
+}
+
+int main(){
+    int num;
+	//take input of integer
+    cin >> num;
+
+    printKeypad(num);
+	//prints all the combinations of the keypad for given input integer
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Phone_keypad.cpp:45]: Display in Print form

File: C-Plus-Plus/other/Phone_keypad.cpp [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    Given an integer number print all the possible combinations of the keypad. You do not need to return anything just print them.
+    */
+    
+    string output = "";
+    printKeypad_rec(num, output);
+}
+
+int main(){
+    int num;
+	//take input of integer
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Phone_keypad.cpp:41]: Display in print form. Also, write complete details, what input ?

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@ishwari20 Please re-verify your program before asking for re-review.
You have syntax error in your first line of code.
Also, do the modifications as suggested.

File: C-Plus-Plus/other/Phone_keypad.cpp [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+Sample output 1:
+da
+ea
+fa
+db
+eb
+fb
+dc
+ec
+fc
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Phone_keypad.cpp]: Sample Input & Output is not matching with Program Execution.
(Print Message is missing in Sample Program)

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

**Review by siddharth25pandey — APPROVED** [review]
@ishwari20 LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@al-contributors please add @ishwari20 for her code

**Comment by siddharth25pandey** [comment]
@all-contributors please add @ishwari20 for her code

**Pull request merged**